pom.xml

```diff
@@ -220,6 +220,18 @@
             <version>2.8.2</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-annotations</artifactId>
+            <version>2.10.1</version>
```

Comment:
Use a property for this and the one below, I think weird things might happen if they don't stay in sync